cmd/car/testdata/script/get.txt

```diff
@@ -0,0 +1,6 @@
+env SAMPLE_CID='bafy2bzaceaycv7jhaegckatnncu5yugzkrnzeqsppzegufr35lroxxnsnpspu'
```

Comment:
following the other files, shouldn't we call this get-dag.txt?

if you prefer get.txt, then I'd join get-block.txt here.

Reply:
the go file under test is `get.go` which has both `get-block` and `get-dag`. what do you think is more reasonable?

Reply:
If you expect the test scripts to grow further, I'd keep them separate